modules/ztp-syncing-new-configmap-changes-to-existing-pgt-crs.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-syncing-new-configmap-changes-to-existing-pgt-crs_{context}"]
+= Syncing new ConfigMap changes to existing PolicyGenTemplate CRs
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have created a `PolicyGenTemplate` CR that pulls information from a `ConfigMap` CR using hub cluster templates.
+
+.Procedure
+
+. Update the contents of your `ConfigMap` CR, and apply the changes in the hub cluster.
+
+. To sync the contents of the updated `ConfigMap` CR to the deployed policy, do either of the following:
+
+.. Optional: Delete the existing policy and recreate it using the `PolicyGenTemplate` CR.
+
+... Delete the policy, for example:
++
+[source,terminal]
+----
+$ oc delete policy <policy_name> -n <policy_namespace>
+----
++
+... Re-apply the `PolicyGenTemplate` CR:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
+----
+
+.. Optional: Apply a special annotation `policy.open-cluster-management.io/trigger-update` to the policy with a different value every time when updating the `ConfigMap`. For example, run the following command:
++
+[source,terminal]
+----
+$ $ oc label <policy_name> -n <policy_namespace> policy.open-cluster-management.io/trigger-update="1"
+----
+
+... Re-apply the `PolicyGenTemplate` CR to update the applied policy. For example, run the following command:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
+----
+
+. Optional: if it exists, delete the `ClusterGroupUpdate` CR that contains the policy. For example:
++
+[source,terminal]
+----
+$ oc delete clustergroupupgrade <cgu_name> -n <cgu_namespace>
+----
+
+.. Create a new `ClusterGroupUpdate` CR that includes the policy to apply with the updated `ConfigMap` changes. For example add the following YAML to the file `cgr-example.yaml`:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: <cgr_name>
+  namespace: <policy_namespace>
+spec:
+  managedPolicies:
+    - <managed_policy>
+  enable: false
+  clusters: <1>
+  - <managed_cluster_1>
+  - <managed_cluster_2>
+  remediationStrategy:
+    maxConcurrency: 2
+    timeout: 240
+----
+<1> The list of clusters to update.
++
+.. Apply the updated policy:
++
+[source,terminal]
+----
+$ oc apply -f cgr-example.yaml
+----
```

Comment:
Probably add this as an additional resource at the end to indicate this limitation in ACM.
https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html-single/governance/index#special-annotation-processing

Reply:
Added as a note inline